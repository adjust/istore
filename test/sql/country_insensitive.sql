SELECT 'de'::country = 'de'::country;
SELECT 'de'::country = 'De'::country;
SELECT 'de'::country = 'dE'::country;
SELECT 'de'::country = 'DE'::country;
SELECT 'dE'::country = 'dE'::country;
SELECT 'dE'::country = 'De'::country;
SELECT 'dE'::country = 'DE'::country;
SELECT 'De'::country = 'De'::country;
SELECT 'De'::country = 'DE'::country;
SELECT 'DE'::country = 'DE'::country;
