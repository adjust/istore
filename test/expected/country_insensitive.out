SELECT 'de'::country = 'de'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'de'::country = 'De'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'de'::country = 'dE'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'de'::country = 'DE'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'dE'::country = 'dE'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'dE'::country = 'De'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'dE'::country = 'DE'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'De'::country = 'De'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'De'::country = 'DE'::country;
 ?column? 
----------
 t
(1 row)

SELECT 'DE'::country = 'DE'::country;
 ?column? 
----------
 t
(1 row)

